build.gradle

```diff
@@ -43,14 +43,19 @@ ext {
   }
   fullVersion = baseVersion + ((!snapshotVersion && releaseCandidate) ? "-RC$releaseCandidate" : "") + "-groovy-$variant" + (snapshotVersion ? "-SNAPSHOT" : '')
   variantLessVersion = baseVersion + (snapshotVersion ? "-SNAPSHOT" : (releaseCandidate ? "-RC$releaseCandidate" : ""))
-  javaVersions = [1.7, 1.8]
+  javaVersions = [1.7, 1.8, 9, 10, 11]
   javaVersion = System.getProperty("java.specification.version") as BigDecimal
+
+  if (javaVersion >= 9) {
+    groovyDependency += ["javax.xml.bind:jaxb-api:2.3.0"]
+  }
+
   libs = [
     jetbrainsAnnotations: "org.jetbrains:annotations:13.0",
     ant: "org.apache.ant:ant:1.9.7",
     asm: "org.ow2.asm:asm:6.2",
     bytebuddy: "net.bytebuddy:byte-buddy:1.8.3",
```

Comment:
There are still new fixed/features related to Java 11 [compatibility](https://github.com/raphw/byte-buddy/releases) in byte-buddy. I propose to upgrade it to the least stable version - 1.8.18.

Reply:
I have tried locally 1.8.17 and some tests fail. I don't believe it is possible due to https://github.com/raphw/byte-buddy/issues/486 

Personally, I would rather make this PR work and then somebody can improve it further and add support for Java 11 (#843). To be honest, I don't anything about the stuff mentioned in #843.